snapshots/health/2024-04-12/polio_free_countries.csv.dvc

```diff
@@ -0,0 +1,26 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: Polio-Free Countries - GPEI
+    date_published: "2016"
```

Comment:
Is it right to say 2016 when there are countries with the `ongoing` category? I see they use the countries as in 2016, but maybe they kept checking and confirming these ongoing statuses

Reply:
Also, because it looks contradictory when we also show data up to 2023

Reply:
True, I will set it to 2023 for now